.eslintrc

```diff
@@ -33,6 +33,14 @@
     }
   },
   "rules": {
+    /* ban-types is split in ESLint v8:
+       https://typescript-eslint.io/rules/ban-types/ */
+    "@typescript-eslint/ban-types": "error",
+    /* The no-useless-constructor needs to be disabled for
```

Comment:
Can we put these in the `*.ts` override then? That way non typed files won't be impacted

Reply:
You're right, fixed!

Reply:
@ljowen in hindsight, I think this was wrong, the typescript-eslint rule extends the regular eslint rule. As far as I can tell, the typescript-eslint rule works as intended for untyped code too.

I have fixed this mistake in #7323.